cli/src/command/config_transaction_create.rs

```diff
@@ -0,0 +1,304 @@
+    let parts: Vec<&str> = action_str.split_whitespace().collect();
+    match parts.get(0).map(|s| *s) {
+        Some("AddMember") => {
+            let new_member = squads_multisig::state::Member {
+                key: Pubkey::from_str(parts.get(1).unwrap()).unwrap(),
+                permissions: Permissions {
+                    mask: parts.get(2).unwrap().parse().unwrap(),
+                },
+            };
+
+            Ok(ConfigAction::AddMember { new_member })
+        }
+        Some("RemoveMember") => {
+            let old_member = parts
+                .get(1)
+                .ok_or("Old member pubkey is required for RemoveMember action")?
+                .parse()
+                .map_err(|_| "Invalid old member pubkey format")?;
+            Ok(ConfigAction::RemoveMember { old_member })
+        }
+        Some("ChangeThreshold") => {
+            let new_threshold = parts
+                .get(1)
+                .ok_or("New threshold is required for ChangeThreshold action")?
+                .parse()
+                .map_err(|_| "Invalid new threshold format")?;
+            Ok(ConfigAction::ChangeThreshold { new_threshold })
+        }
+        Some("SetTimeLock") => {
+            let new_time_lock = parts
+                .get(1)
+                .ok_or("New time lock is required for SetTimeLock action")?
+                .parse()
+                .map_err(|_| "Invalid new time lock format")?;
+            Ok(ConfigAction::SetTimeLock { new_time_lock })
+        }
+        Some("AddSpendingLimit") => parse_add_spending_limit(&parts[1..]),
+        Some("RemoveSpendingLimit") => {
+            let spending_limit = parts
+                .get(1)
+                .ok_or("Spending limit pubkey is required for RemoveSpendingLimit action")?
+                .parse()
+                .map_err(|_| "Invalid spending limit pubkey format")?;
+            Ok(ConfigAction::RemoveSpendingLimit { spending_limit })
+        }
+        Some("SetRentCollector") => {
+            let new_rent_collector = parts
+                .get(1)
+                .map(|s| s.parse())
+                .transpose()
+                .map_err(|_| "Invalid rent collector pubkey format")?;
+            Ok(ConfigAction::SetRentCollector { new_rent_collector })
+        }
+        _ => Err("Invalid or unsupported action".to_string()),
+    }
+}
+
+fn parse_add_spending_limit(parts: &[&str]) -> Result<ConfigAction, String> {
+    if parts.len() < 7 {
+        return Err("Not enough arguments for AddSpendingLimit".to_string());
+    }
+
+    fn parse_pubkey_list(list_str: &str) -> Result<Vec<Pubkey>, String> {
+        let mut pubkeys = Vec::new();
+        for s in list_str.split(',') {
+            let pubkey = s.parse().map_err(|_| "Invalid pubkey")?;
+            pubkeys.push(pubkey);
+        }
+        Ok(pubkeys)
+    }
+
+    let create_key = parts[0].parse().map_err(|_| "Invalid create_key format")?;
```

Comment:
Do I understand correctly that we're relying on the order of arguments here? If so this seems quite error prone for the user,I would try nested fields here instead, smth like anchor's --provider.wallet, --provider.cluster

Reply:
not needed for now as documentation is clear (copy paste)